cw/cw305/pyXKCP/pyXKCP_extention_build.py

```diff
@@ -0,0 +1,95 @@
+#int KMAC256_Final(KMAC_Instance *kmkInstance, BitSequence *output);
+#int KMAC256_Squeeze(KMAC_Instance *kmkInstance, BitSequence *output, BitLength outputBitLen);
+#int ParallelHash128( const BitSequence *input, BitLength inputBitLen, size_t blockByteLen, BitSequence *output, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int ParallelHash128_Initialize(ParallelHash_Instance *ParallelHashInstance, size_t blockByteLen, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int ParallelHash128_Update(ParallelHash_Instance *ParallelHashInstance, const BitSequence *input, BitLength inputBitLen);
+#int ParallelHash128_Final(ParallelHash_Instance *ParallelHashInstance, BitSequence * output);
+#int ParallelHash128_Squeeze(ParallelHash_Instance *ParallelHashInstance, BitSequence *output, BitLength outputBitLen);
+#int ParallelHash256( const BitSequence *input, BitLength inputBitLen, size_t blockByteLen, BitSequence *output, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int ParallelHash256_Initialize(ParallelHash_Instance *ParallelHashInstance, size_t blockByteLen, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int ParallelHash256_Update(ParallelHash_Instance *ParallelHashInstance, const BitSequence *input, BitLength inputBitLen);
+#int ParallelHash256_Final(ParallelHash_Instance *ParallelHashInstance, BitSequence * output);
+#int ParallelHash256_Squeeze(ParallelHash_Instance *ParallelHashInstance, BitSequence *output, BitLength outputBitLen);
+#int TupleHash128( const TupleElement *tuple, size_t numberOfElements, BitSequence *output, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int TupleHash128_Initialize(TupleHash_Instance *TupleHashInstance, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int TupleHash128_Update(TupleHash_Instance *TupleHashInstance, const TupleElement *tuple, size_t numberOfElements);
+#int TupleHash128_Final(TupleHash_Instance *TupleHashInstance, BitSequence * output);
+#int TupleHash128_Squeeze(TupleHash_Instance *TupleHashInstance, BitSequence *output, BitLength outputBitLen);
+#int TupleHash256( const TupleElement *tuple, size_t numberOfElements, BitSequence *output, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int TupleHash256_Initialize(TupleHash_Instance *TupleHashInstance, BitLength outputBitLen, const BitSequence *customization, BitLength customBitLen);
+#int TupleHash256_Update(TupleHash_Instance *TupleHashInstance, const TupleElement *tuple, size_t numberOfElements);
+#int TupleHash256_Final(TupleHash_Instance *TupleHashInstance, BitSequence * output);
+#int TupleHash256_Squeeze(TupleHash_Instance *TupleHashInstance, BitSequence *output, BitLength outputBitLen);
+
+ffibuilder.set_source("_XKCP",  # name of the output C extension #ToDo: better naming?
```

Comment:
I don't really know about the naming "_XKCP" sounds reasonable to me. Maybe the [Python style guide](https://docs.opentitan.org/doc/rm/python_coding_style/) has some guidance on that? 

In fact, the [PEP 8 Style Guide](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) says:
> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.
>
> When an extension module written in C or C++ has an accompanying Python module that provides a higher level (e.g. more object oriented) interface, the C/C++ module has a leading underscore (e.g. _socket).

So, probably `_xkcp` should be used.

Reply:
done